public/static/docs/command-reference/import.md

```diff
@@ -160,23 +159,14 @@ deps:
 If `rev` is a Git branch or tag (where the underlying commit changes), the data
 source may have updates at a later time. To bring it up to date if so (and
 update `rev_lock` in the DVC-file), simply use `dvc update <stage>.dvc`. If
-`rev` is a specific commit hash (does not change), `dvc update` will never have
-an effect on the import stage. You may **re-import** a different commit instead,
-by using `dvc import` again with a different (or without) `--rev`. For example:
+`rev` is a specific commit (does not change), `dvc update` will never have an
+effect on the import stage. You may `dvc update` to a different commit, using
+`--rev`:
 
 ```dvc
-$ dvc import --rev master \
-             git@github.com:iterative/dataset-registry.git \
-             use-cases/cats-dogs
+$ dvc update --rev cats-dogs-v2
```

Comment:
```suggestion
`rev` is a specific commit hash (does not change), `dvc update` without options will not have an
effect on the import stage. You may force-update it to a different commit with `dvc update --rev`:

```dvc
$ dvc update --rev master
```
I prefer master here because it's a more common case to go to the latest version, I think.

Reply:
Yeah, it looks good, except the snippets. I'd like to keep code as it is because we would be moving between tags and branches as mentioned  on https://github.com/iterative/dvc.org/pull/1007#discussion_r380778856.

And, this should be more about `import`, not `update`.

Reply:
No strong opinion but I don't see the problem in going from a tag to master. Going from anywhere to master is probably one of the most common use cases.